README.md

```diff
@@ -1,8 +1,9 @@
+# Income Tax (Making Tax Digital) Changelog
 
-# income-tax-mtd-changelog
 
-The following Income Tax MTD services are supported:
+### List of services
```

Comment:
Doesn't really need the heading as it's the introduction

Reply:
Updated as per feedback, can you review again?